bl-kernel/boot/rules/60.plugins.php

```diff
@@ -78,6 +78,9 @@ function buildPlugins()
 		$languageFilename = PATH_PLUGINS.$Plugin->directoryName().DS.'languages'.DS.$site->language().'.json';
 		if( !Sanitize::pathFile($languageFilename) ) {
 			$languageFilename = PATH_PLUGINS.$Plugin->directoryName().DS.'languages'.DS.DEFAULT_LANGUAGE_FILE;
+			if( !Sanitize::pathFile($languageFilename) ) {
+				continue;
+			}
```

Comment:
Hi, I don't get this condtion here again ?

Reply:
I know what you said, but I'm trying to keep performance and not add a lot of checks (these lines are executed for each user who visit the site), the checks are good but are not 100% need it, that why I recommend to the users to keep a structure of files when they create a plugin or theme.

if the administrator set anything in the constant `DEFAULT_LANGUAGE_FILE` is his fault.

I don't know if you understand my point, is not against to you, I'm trying to not create a bloated software.

Reply:
I understand you and you're right, performance is way more important then to prevent each possible error or administration mistake, especially in this case.